content/news/2020-09-11-system-sound-contest.md

```diff
@@ -0,0 +1,65 @@
++++
+type = "blog"
+title = "Contest for System Sounds"
+author = "bitigchi"
+date = "2020-09-06 13:00:00+03:00"
+tags = ["sound", "contest"]
++++
+
+It wouldn't be wrong to say that, Haiku is a unique operating system. It takes its roots from BeOS, the OS everyone envied back in the day. Haiku continues to follow its footsteps, trying to deliver that same experience. What could be better than complementing this unique operating system with a set of unique system sounds?
+
+As R1 steadily approaches, we are looking for contestants (amateur, professional, enthusiast) to produce system sound effects for Haiku. Specifically sounds for:
+
+## Sound Events
+
+**Errors:**
+
+- Beep
+- Error notification
+
+**Notifications:**
+
+- Important notification
+- Information notification
+- Progress notification
+
+**Key/mouse events:**
+
+- Key down
+- Key repeat
+- Key up
+- Keymap switch
+- Mouse down
+- Mouse up
+
+**Window events:**
+
+- Window activated
+- Window close
+- Window minimised
+- Window open
+- Window restored
+- Window zoomed
+
+- In addition to these above, a nice soothing **startup chime**
+
+## Contest guidelines
+
+- All sound effects except the startup chime should be up to 1 second long. If the situation calls for, it can be longer, but it **should** not exceed 2 seconds. For startup chime, the limit is 5 seconds.
+- All sound effects should be non-intrusive. Distorted tunes, high-pitched tunes will not be considered.
+- Expanding on "haiku" and related themes are encouraged.
+- Entries need to be in WAV format (44,100 Hz, 16 bits ps).
+- Entries should include a note that the work is licensed under [CC Attribution-ShareAlike 4.0 International](https://creativecommons.org/licenses/by-sa/4.0/) or a similar license.
+- Naming the sound files according to the theme is encouraged, but intended sound event should be noted appropriately.
+
+## Submission deadline
+
+Deadline for the contest is December 1st, 2020; we would like to ship the new default sound theme for Haiku with Beta3. We also plan to package and release qualifying submissions as separate packages available for download on the main Haiku software repository.
+
+## How to participate
+
+Send an e-mail to humdingerb (gmail) com and bitigchi (me) com with a link to your zipped up work. Entries and news regarding the contest will be added to this post.
+
+The winner will be announced on the website and the Haiku general mailing list. Modifications will be accepted until November 27th, 2020.
```

Comment:
Should we include a hint at the deciding poll forum post (and link to it once it's up)? Like:

```suggestion
The winner will be decided by a poll in the forums, and will be announced on the website and the Haiku general mailing list. Modifications will be accepted until November 27th, 2020.
```
Otherwise, I think we're good to go.